dashboard/test/ui/features/star_labs/maze.feature

```diff
@@ -27,13 +27,13 @@ Scenario: Submit an invalid solution
 Scenario: Submit a valid solution
   When I wait for the page to fully load
   Then element "#resetButton" is hidden
-  Then I drag block "4" to block "6"
-  And I drag block "1" to block "7" plus offset 35, 50
-  Then block "8" is child of block "7"
-  Then I drag block "5" to block "8"
-  And block "9" is child of block "8"
-  Then I drag block "3" to block "9" plus offset 35, 30
-  And block "10" is child of block "9"
+  Then I drag block "repeatForever" to block "topBlock"
+  And I drag block "moveForward" into first position in repeat block "repeatForever"
```

Comment:
Not sure why we weren't using this step previously, but it's equivalent and more readable.